examples/canvas-drawing.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title>perf.html example &mdash; Canvas Drawing</title>
+    <link rel='stylesheet' href='examples.css'>
+  </head>
+  <body>
+    <h1>perf.html example &mdash; Canvas Drawing</h1>
+    <p>
+      This example shows the performance impacts of drawing rectangles using canvas
+      context calls. It shows how the CSS parser is used when setting the colors for
+      fillStyle, as well as what requestAnimationFrame loops look like.
+    </p>
+    <p>
+      <label for='number'>Rectangles:</label>
+      <input id='number' type='number' value='10000' />
+    </p>
+    <p>
+      <button id='toggle'>Toggle mixed colors</button>
+    </p>
+
+    <div id='container'>
+      <canvas />
```

Comment:
Is `<canvas />` a thing? I thought in HTML5 this needed to be `<canvas></canvas>`.